.github/workflows/deploy_on_ec2.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         ARGS: "-rlgoDzvc -i --delete"
         SOURCE: "target/"
         TARGET: "/home/ec2-user/calendar-backend"
+        EXCLUDE: "/debug/build, /debug/deps/, /debug/examples/, /debug/incremental/, /debug/.fingerprint/"
```

Comment:
Excluded the uploading of unnecessary artifacts to EC2. This helps by making the deployment a little faster.